admin/tabs/configs/differ/ConfigDifferModel.js

```diff
@@ -82,10 +84,12 @@ export class ConfigDifferModel  {
     }
 
     async loadAsync() {
+        const remoteHost = `${trimEnd(this.remoteHost, '/') + (XH.baseUrl == '/api/' ? '/api' : '/')}`;
```

Comment:
I'm dubious about this. Not sure if nginx will ever change how it changes the backend urls but this seems pretty hard coded. That to one side, I would rather we had a XH.isDeployed (or some variation) to check for this switch which would at least make the code more semantic.

Even better a XH.apiSuffix which could be set to an empty string in development.